### PR TITLE
Added samples for the kotlin.system package

### PR DESCRIPTION
**Pull request by uzilan**
Added samples for the kotlin.system package

Issue: https://youtrack.jetbrains.com/issue/KT-20357

### CODE REVIEW

File: libraries/stdlib/samples/test/samples/system/timing/timing.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import samples.*
+import kotlin.system.*
+import kotlin.test.*
+
+class Timings {
+
+    @Sample
+    fun measureBlockTimeMillis() {
+        val timeInMillis = measureTimeMillis {
+            (0..10).reduce(Int::plus)
```
Comment (ilya-g) [inline libraries/stdlib/samples/test/samples/system/timing/timing.kt:12]: I suppose given the resolution of millisecond timer this result is going to be 0 every time. Perhaps we can find some other task that lasts for 10-20 ms in Kotlin playground?

File: libraries/stdlib/samples/test/samples/system/timing/timing.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        }
+        assertTrue(timeInMillis >= 0)
+        assertTrue(timeInMillis < 1e9)
```
Comment (ilya-g) [inline libraries/stdlib/samples/test/samples/system/timing/timing.kt:15]: I think such assertions are not that informative for the purpose of sample. Would be better just to print out the value, something like `println("The operation took $timeInMillis ms")`

File: libraries/stdlib/samples/test/samples/system/timing/timing.kt (hunk 1 above)
Comment (uzilan) [reply libraries/stdlib/samples/test/samples/system/timing/timing.kt:12]: Cool. I'll try to find something else to use.

File: libraries/stdlib/samples/test/samples/system/timing/timing.kt (hunk 2 above)
Comment (uzilan) [reply libraries/stdlib/samples/test/samples/system/timing/timing.kt:15]: Good suggestions. I'll try that.

**Comment by ilya-g** [comment]
Thanks, I've merged it manually with some corrections

**Pull request closed without merging**